### PR TITLE
Bridge Cardax testnet launch work PR

### PR DESCRIPTION
**Pull request by bladyjoker**

### CODE REVIEW

File: roundtrip/RoundTrip/Spec.hs [hunk 1]
```diff
@@ -146,7 +147,10 @@ roundTripSpec = do
       -- IPC
       hPutStrLn hin jsonReq
       err <- hGetLine herr
-      assertEqual "hs> Purescript shouldn't report an error" "" err
+      when (err /= "") $ do
```
Comment (ngua) [inline roundtrip/RoundTrip/Spec.hs:153]:
```suggestion
      unless (null err) $ do
```

File: roundtrip/RoundTrip/Spec.hs [hunk 2]
```diff
@@ -118,17 +118,18 @@ roundTripSpec = do
               -- IPC
               resp <- doReq hin herr hout (Req RTPlutusData payload)
               -- Assert response
-              pdResp <-
+              payload' <-
                 response
                   (\err -> assertFailure $ "hs> Wanted ResSuccess got ResError " <> err)
```
Comment (ngua) [inline roundtrip/RoundTrip/Spec.hs]: You could have a helper like `assertFailure' = asssertFailure . ("hs>" <>)` to factor out that part

**Review by ngua — APPROVED** [review]

File: roundtrip/RoundTrip/Spec.hs (hunk 2 above)
Comment (bladyjoker) [reply roundtrip/RoundTrip/Spec.hs]: Done

File: roundtrip/RoundTrip/Spec.hs (hunk 1 above)
Comment (bladyjoker) [reply roundtrip/RoundTrip/Spec.hs:153]: I prefer the `when`, somehow `unless` breaks my feeble mind

**Pull request merged**